handlers/organizations.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const Formatters = require('../lib/formatters');
+
+function errorHandler(reply, e) {
+    return this.helpers.errorHandler.apply(this, arguments);
+}
+
+// GET /organizations
+exports.listOrganizations = async function(request, reply) {
+    try {
+        const organizations = await this.models.organization.listOrganizations();
```

Comment:
The model always begins with a capital letter. Here it will be `this.models.Organization.listOrganizations()`